### DISCUSSION TITLE
UI - Make GitHub invisible

### DISCUSSION THREAD

**Discussion by sethdaily** [tinacms/tinacms#4478] [Ideas]
For content editors, the process is confusing. You have to create a branch, make edits, and open GitHub in a new window to create a pull request and merge it.

Solution:

- Git workflow is behind the scenes
- User doesn't see branches or Pull Requests

**Comment by Calinator444** [comment]
I might be biased here as a dev but I think this is more of a nice to have. There is a steep learning curve for git if you're not a developer, but the way Tina works now is acceptable for an MVP.

**Reply by eduard-nz** [reply]
Is this as a highly valuable feature. Have seen the mock ups in the YouTube video, and this would make a lot of editors super happy

**Comment by SamBWagner** [comment]
I'm not sure on who the primary consumer of Tina is. So this is either essential or barely worth doing. We should identify our consumers first.
